runatlantis.io/contributing.md

```diff
@@ -1,3 +1,6 @@
+---
+aside: false
```

Comment:
these hides the right nav; those page looks better without them, and it only has a single item there anyway

Reply:
make sense 👍